### PR TITLE
add pycld2

### PR DESCRIPTION
**Pull request by FernandezMathieu**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/pycld2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - python >=2.7
+
+test:
+  imports:
+    - pycld2
+  commands:
+    - pip check
+  requires:
+    - pip
+    - wheel==0.23.0
```
Comment (dopplershift) [inline recipes/pycld2/meta.yaml:32]: Any reason this can't just be `wheel`?

File: recipes/pycld2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycld2-{{ version }}.tar.gz
+  sha256: a42f6e974df8fdd70685c2baa8a9f523069a260e1140ce604fb9f1fb6c3064df
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
```
Comment (dopplershift) [inline recipes/pycld2/meta.yaml:19]: This looks like it builds C++ files, so I'm pretty sure somewhere you'll need a:
```yaml
    build:
        - compiler('cxx')
```

File: recipes/pycld2/meta.yaml (hunk 1 above)
Comment (FernandezMathieu) [reply recipes/pycld2/meta.yaml:32]: It's stange for me too but the requirement file in the project's repository got only wheel in a specific version https://github.com/aboSamoor/pycld2/blob/master/requirements.txt

Comment (FernandezMathieu) [reply recipes/pycld2/meta.yaml]: Trying with `wheel >= 0.23.0`

File: recipes/pycld2/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  sha256: a42f6e974df8fdd70685c2baa8a9f523069a260e1140ce604fb9f1fb6c3064df
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - compiler('cxx')
```
Comment (dopplershift) [inline recipes/pycld2/meta.yaml:20]: Oops, apparently needs to be templated, so should be:
```suggestion
    - {{ compiler('cxx') }}    
```

**Comment by FernandezMathieu** [comment]
@dopplershift need some advice here, i can only build in linux

**Comment by FernandezMathieu** [comment]
@conda-forge/core hello, i hustle to build this package, someone can help me ?

File: recipes/pycld2/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python >=2.7
+  run:
+    - python >=2.7
+    - wheel >=0.23.0
+    - libstdcxx-ng
+    - libgcc-ng
```
Comment (isuruf) [inline recipes/pycld2/meta.yaml:28]:
```suggestion
```

Comment (dopplershift) [reply recipes/pycld2/meta.yaml:28]: I think it was complaining about not finding libstdc++ (or something) before those were added.

Comment (FernandezMathieu) [reply recipes/pycld2/meta.yaml]: Build linux fail with this message : 
`but ['libstdcxx-ng'] not in reqs/run, [...] ['libgcc-ng'] not in reqs/run`

File: recipes/pycld2/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycld2-{{ version }}.tar.gz
+  sha256: a42f6e974df8fdd70685c2baa8a9f523069a260e1140ce604fb9f1fb6c3064df
+
+build:
+  number: 0
+  noarch: python
```
Comment (isuruf) [inline recipes/pycld2/meta.yaml:15]:
```suggestion
```

Comment (dopplershift) [reply recipes/pycld2/meta.yaml:15]: 🐑 How did I not catch that?

Comment (FernandezMathieu) [reply recipes/pycld2/meta.yaml]: I removed that, but still can't be able to build

**Comment by dopplershift** [comment]
Looks like the compiler just doesn't like something in the code. Not sure if there's some option that should make it build. This is a Google project (it seems) so I'd expect it to compile fine with clang.

**Pull request closed without merging**